faqs/README.md

```diff
@@ -0,0 +1,72 @@
+# FAQs
```

Comment:
I see it is part of a tutorial. We need to keep in mind to update both location if there is any changes